apps/full-site-editing/full-site-editing-plugin/site-editor/index.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Site Editor experiment file.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+/**
+ * Enables/Disables site editor experiment per blog sticker.
+ */
+function initialize_site_editor() {
+	if ( ! is_site_editor_active() ) {
+		return;
+	}
+
+	// Force enable required Gutenberg experiments if they are not already active.
+	add_filter(
```

Comment:
It's still a bit confusing to me why this is required, mainly since you still have to toggle the checkmark in the experiments options. I think we should definitely add some comments clarifying why we need this.

I mean, it definitely doesn't work when I comment these lines out. Since the basic DOM of the site editor is still loaded without these lines, I guess the error happens when the JS is starting to do stuff. So maybe this is where the public-api comes into play? I guess some request is failing which doesn't think that the experiment is enabled.